axum-extra/src/routing/spa.rs

```diff
@@ -150,8 +150,8 @@ impl<B, T, F> SpaRouter<B, T, F> {
 impl<B, F, T> From<SpaRouter<B, T, F>> for Router<B>
 where
     F: Clone + Send + 'static,
-    HandleError<Route<B, io::Error>, F, T>:
-        Service<Request<B>, Response = Response, Error = Infallible>,
+    HandleError<Route<B, io::Error>, F, T>: Service<Request<B>, Error = Infallible>,
+    <HandleError<Route<B, io::Error>, F, T> as Service<Request<B>>>::Response: IntoResponse + Send,
```

Comment:
It's interesting that in a few cases like this one, `Send` is also required while in others it isn't. Did you investigate why that is? Should we maybe have it everywhere for consistency and allow a little more leeway with implementations?